minecraft_servers/cubic/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+    "server_name": "cubic",
+    "nice_name": "Cubic",
+    "direct_ip": "djhase.de",
+    "supported_languages": [
+      "de", "en"
+    ],
+    "social": {
+      "web": "https://djhase.de",
+      "discord": "https://dc.djhase.de"
```

Comment:
Custom Discord invites are reserved for partners only. Set a normal Discord invite link: https://discord.gg/[YOUR-INVITE]